vfs/vfstest/fs.go

```diff
@@ -310,7 +313,7 @@ func writeFile(filename string, data []byte, perm os.FileMode) error {
 
 func (r *Run) createFile(t *testing.T, filepath string, contents string) {
 	filepath = r.path(filepath)
-	err := writeFile(filepath, []byte(contents), 0600)
```

Comment:
Why do you need this change? Is it something to do with the NFS server?

Reply:
When the file is read again, it is always 0644, regardless of the mode that is passed in. NFS tries to do a chmod if the mode is not correct and it would fail.